posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+Download the `pkg` file that is appropriate for your operating system (_i.e._, 
+`napari-0.4.17-macOS-x86_64.pkg` for Mac). Open the downloaded file and install 
+via the instructions on screen (you can accept the defaults). When finished, 
+launch napari via the Applications folder. 
+
+> Pro tip: If launching for the first time, the application may take a moment to appear. Avoid launching multiple instances. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-pkg-download"
+#| fig-cap: "Screenshot of napari packages. Blue highlighted package is appropriate for Mac. Other operating systems' packages are also available but untested."
+
+knitr::include_graphics("./figures/mac_install_pkg.png")
+```
+
+
+
+#### Part 2: Installing `napari-cosmx`
+
+1. In napari, open up the `>_` button that is located on the bottom left (see @fig-mac-plugin-command for example). 
+2. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+3. Paste the location of the `whl` file
+4. Press enter to execute
+5. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+6. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
+
+knitr::include_graphics("./figures/mac_plugin_command.png")
+```
+
+:::
+
+
+## Napari-ready slide folder {#sec-napari-ready-files}
+
+The `napari-cosmx` plugin expects a single slide that has been prepared using the
+`stitching` (@sec-stitching) method within the plugin itself. If you do not already
+have a napari-ready slide, you can download a two-FOV example that we have created
+[here (to do: need to link the two-fov file)](){target="_blank"}. This two-FOV dataset
+was derived from the coronal hippocampus and
+cortex public dataset. The full data download is not needed for our current purposes but
+those interested in exploring the full data can download it  [here](https://nanostring.com/resources/coronal-hippocampus-and-cortex-raw-data-export/){target="_blank"}). 
+
+At the very minimum, the napari-ready slide folder contains two elements: 
+
+1. `images` folder. Within the `images` folder, there are subfolders for each Immunofluorescence channel and one folder for the cell boundaries (`labels`). 
+2. `targets.hdf5` file that contains the RNA (or protein) targets. 
+
+And example can be see below (@fig-napari-ready-folder).
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-napari-ready-folder"
+#| fig-cap: "Example layout of a napari-ready folder. In this example, the 'parent folder' (_i.e._, the folder you would drag and drop into napari) is named `two_fovs_napari_files` and contains the `images` folder with subfolders containing various zarr files within and the `targets.hdf5` file."
```

Comment:
zarr is only ever mentioned here. Either introduce this file type earlier when discussing napari formats generally or remove the mention here. It's in the description on the widget, so preferable would be to describe this format in earlier section.

Reply:
I added a brief description and link to it in the `What is the napari-cosmx plugin` subsection. 
2c07f5b40f6bc1f27aaa6391ed017a61ec807857